workers/unity/Packages/com.improbable.gdk.playerlifecycle/Config/PlayerLifecycleConfig.cs

```diff
@@ -13,11 +13,11 @@ namespace Improbable.Gdk.PlayerLifecycle
     ///     serialized data when calling `RequestPlayerCreation` manually.
     /// </param>
     /// <remarks>
-    ///     This is called by the `HandleCreatePlayerRequestSystem` to get a player entity template
+    ///     This is called by the <see cref="HandleCreatePlayerRequestSystem"/> to get a player entity template
```

Comment:
This isn't called - its the type of the function. 

```
public static GetPlayerEntityTemplateDelegate CreatePlayerEntityTemplate;
```

is the one which is called.